src/lib/modernExtend.ts

```diff
@@ -454,6 +455,12 @@ export function battery(args?: BatteryArgs): ModernExtend {
         result.meta = meta;
     }
 
+    if (args.lowStatusReportingConfig) {
+        const configure: Configure[] = result.configure || [];
```

Comment:
```js
const result: ModernExtend = {exposes, fromZigbee, toZigbee, isModernExtend: true};
result.configure = []
```

to prevent creating a configure here and under `percentageReporting`

Reply:
done